data/governments.txt

```diff
@@ -858,6 +858,8 @@ government "Remnant"
 		capture 10
 		destroy 10
 		scan 0.1
+	"use foreign penalties for"
```

Comment:
In generall i would split data file changes from code changes but i guess this one is ok

Reply:
Its a fix and its also to test the PR out
I could but Z is busy too so it wouldnt get fixed that fast